### PR TITLE
orchestra: serve psiphon config via authenticated API

### PR DESCRIPTION
**Pull request by bassosimone**
This PR implements serving psiphon config via authenticated API. The design I've chosen is such that we can eventually have more than one psiphon config inside the table. This way, we could in principle serve one of them at random. For now, however, we serve the first config we find.

I chose to put the verbatim JSON inside the database, because I do not see a specific use case where we want to parse/serialize this config, and therefore I think this would only add complexity.

This PR should not be squashed and merged, since I added also a couple of yak shaving commits addressing issues that I noticed while getting myself acquainted with orchestra.

### CODE REVIEW

File: registry/ooni-registry.toml.example [hunk 1]
```diff
@@ -4,10 +4,14 @@ log-level = "debug"
...
 
 [api]
 port = 8080
 address = "127.0.0.1"
 
 [database]
 url = "postgres://user:pass@hostname:port/dbname?sslmode=disable"
+active-probes-table = "active_probes"
+probe-updates-table = "probe_updates"
+accounts-table = "accounts"
```
Comment (hellais) [inline registry/ooni-registry.toml.example]: These config file variables are not actually used.

File: orchestrate/orchestrate/handler/test_lists.go [hunk 2]
```diff
@@ -190,3 +192,15 @@ func URLsHandler(c *gin.Context) {
...
+func PsiphonConfigHandler(c *gin.Context) {
+	content, err := ioutil.ReadFile(viper.GetString("psiphon.config-file"))
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, gin.H{
+			"error": "server side error",
+		})
+		return
+	}
+	c.Data(http.StatusOK, "application/json", content)
+}
```
Comment (hellais) [inline orchestrate/orchestrate/handler/test_lists.go]: Looks good to me! 👍

File: registry/ooni-registry.toml.example (hunk 1 above)
Comment (bassosimone) [reply registry/ooni-registry.toml.example]: ah, okay, will remove them from the diff then, thank you!

File: registry/ooni-registry.toml.example [hunk 3]
```diff
@@ -4,10 +4,14 @@ log-level = "debug"
 
 [auth]
 jwt-secret = "CHANGEME (must be in sync amongst all instances using JWT)"
+admin-password = "CHANGEME"
```
Comment (hellais) [inline registry/ooni-registry.toml.example]: This is used, so it's a good idea to keep it in the example.

**Pull request merged**